packages/schemas/tables/custom_ui_assets.sql

```diff
@@ -0,0 +1,12 @@
+create table custom_ui_assets (
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  id varchar(21) not null,
+  files jsonb /* @use UploadFiles */ not null default '[]'::jsonb,
+  created_at timestamptz not null default(now()),
+  updated_at timestamptz not null default(now()),
```

Comment:
i don't think the user can update once the file is uploaded

Reply:
ok

Reply:
Table removed